README.md

```diff
@@ -56,17 +56,21 @@ Later, named scenes represent customer-specific applications or demos that we su
-| scene10      | Move a large distance around the robot base                                                                              | UR5      |
-| scene11      | Specific start-goal pose pairing that has been found to induce collisions when used with the built-in UR5 planning       | UR5      |
-| scene12      | Specific start-goal pose pairing that is difficult to execute with any low-level robot motion primitive other than MoveJ | UR5      |
+| Basic Scenes | Description of Task                                                                                                      | Arm + EE Used |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| scene1       | Straight-line move in unrestricted space                                                                                 | UR5 + None    |
+| scene2       | Straight-line move with large obstacles nearby                                                                           | UR5 + None    |
+| scene3       | Reach over a short obstacle that obstructs the direct path                                                               | UR5 + None    |
+| scene4       | Reach over a short obstacle that obstructs the direct path                                                               | xArm6 + None  |
+| scene5       | Reach through a window with a large wall on one side                                                                     | xArm7 + None  |
+| scene6       | Reach through a window with a large wall behind and to the side                                                          | xArm7 + None  |
+| scene7       | Reach over a short obstacle, staying within a narrow corridor                                                            | xArm6 + None  |
+| scene8       | Pouring motion with end effector, with large obstacles nearby                                                            | xArm7 + None  |
+| scene9       | Large arm motion within a “forest”, numerous small obstacles in a random distribution surrounding the robot              | UR5 + None    |
+| scene10      | Move a large distance around the robot base                                                                              | UR5 + None    |
+| scene11      | Specific start-goal pose pairing that has been found to induce collisions when used with the built-in UR5 planning       | UR5 + None    |
+| scene12      | Specific start-goal pose pairing that is difficult to execute with any low-level robot motion primitive other than MoveJ | UR5 + None    |
+
+
+| User Scenes   | Description of Task                                                                                                     | Arm + EE Used |
+| 13: objSearch | Based on a user application involving a UR5e and a Viam gripper tasked with moving to random poses in a workspace       | UR5 + VG      |
```

Comment:
q: do we want to note these separately? If this is the case maybe we should use a different numbering system, since adding more to the first would cause us to have to renumber everything?

Reply:
I thought about that, I don't really like the numerical ordering approach and would prefer a `map[string]func()` here, but wanted to talk about that with the group. I can hold on changing that until Peter gets back so we all have some input but that's my preference.

Reply:
seconding `map[string]func()`

Reply:
The problem I see with strings is that we're going to end up with names like "move-UR-arm-with-orientation-change-over-wall" and I'm not sure thats better than just calling it "4" with a comment attached :P

Reply:
Yeah, seems like a few options but all have drawbacks. It would be awesome to have those keys enumerated in the results tabulation too! But then the distinctiveness you mentioned might cause those tables to get huge. I think we can find a good compromise, but the numbering is my least favorite so far.

Reply:
Ended up not changing this formatting.